x-pack/plugins/enterprise_search/server/routes/workplace_search/groups.ts

```diff
@@ -245,7 +245,7 @@ export function registerBoostsGroupRoute({
   );
 }
 
-export function registerWSGroupRoutes(dependencies: IRouteDependencies) {
+export function registerGroupRoutes(dependencies: IRouteDependencies) {
   registerGroupsRoute(dependencies);
   registerSearchGroupsRoute(dependencies);
   registerGroupRoute(dependencies);
```

Comment:
@scottybollinger I wasn't sure if you wanted to move all these registrations to the new index.ts or keep them grouped in this file for simplicity - let me know if you have a preference and I can make that change in this PR

Reply:
Moving it is fine. Thanks! Will definitely have an effect on https://github.com/elastic/kibana/pull/82669 but I can make changes after we merge this or vice versa.

Reply:
Sweet, thanks Scotty! I'll move them here shortly.

Reply:
Moved in ac5ddaa - hope I got it right, feel free to tweak further in #82669 if I didn't